ui/app/pages/send/send-content/add-recipient/ens-input.component.js

```diff
@@ -215,7 +215,7 @@ export default class EnsInput extends Component {
                 this.props.scanQrCode()
               }
             }}
-          />
+          ></button>
```

Comment:
It looks like the close tag was left here accidentally - I think this will fix the lint error as well:
```suggestion
          />
```